.github/workflows/github-badges.yml

```diff
@@ -0,0 +1,51 @@
+name: Generate API Documentation
+on:
+  push:
+    branches:
+      - main
+jobs:
+  sync-badges-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: badges
+          fetch-depth: 0
+      - name: Sync badges branch with main branch
+        run: |
+          git config user.name 'github-actions[bot]'
+          git config user.email '41898282+github-actions[bot]@users.noreply.github.com'
+          git merge origin/main
+          git push
+  generate-and-update-coverage-badge:
+    needs: sync-badges-branch
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code from branch badges
+        uses: actions/checkout@v2
+        with:
+          ref: badges
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```

Comment:
In gradle we use Java Version 17

Reply:
I just copied that :see_no_evil: updated